config_example.json

```diff
@@ -2,6 +2,17 @@
  "name": "Skynet",
  "encryptConfig": 0,
  "globalHTTPTimeout": 15000000000,
+ "database": {
+  "enabled": false,
+  "driver": "sqlite",
+  "connectionDetails": {
+   "Host": "",
+   "Port": 0,
+   "Username": "",
+   "Password": "",
+   "Database": ""
+  }
```

Comment:
Should this also now have an SSL mode?

Reply:
Yes I left it out to test you and see if you were really reviewing.... that is my story anyway and I am sticking to it 🤥